### ISSUE TITLE
Create a website for current / historical JEPs

### ISSUE THREAD

**Issue by choldgraf** [jupyter/enhancement-proposals#51]
We recently created a little jupyter book for the governance documentation (https://jupyter.org/governance/). The goal was to make the information more discoverable, easier to browse, etc.

What do folks think about doing the same for the enhancement proposals? I really like the idea of these structured proposals, and especially with the meta-JEP that @captainsafia put together I'd like that information to be more discoverable.

So what do people think about a website with two parts:

1. Information about JEPs (background, guides for the JEP process, reusing the meta-JEP language, etc.
2. A list of past and present JEPs, broken down by "inactive" and "active".

This could be hosted at jupyter.org/enhancement-proposals, and could be a way to give more visibility to this process. What do folks think?

**Comment by Zsailer** [comment]
I **love** this idea.

**Comment by blink1073** [comment]
![image](https://user-images.githubusercontent.com/2096628/83059303-5349d780-a01f-11ea-92a8-aec90f849e02.png)

**Comment by captainsafia** [comment]
Sounds like a good idea overall.

We might be able to build upon the UI that Python.org uses for PEPs. @willingc might have more insight here.

**Comment by willingc** [comment]
In general, I think it's a good idea. I think at minimum that there should be:

- a repo that is the source of truth for JEPs. 
- that repo should be used for rendering of any website content
- do we have a process similar to PEP 1 about purpose and guidelines as well as [well written PEPs](https://www.python.org/dev/peps/pep-0001/#id46)

**Comment by choldgraf** [comment]
Some quick thoughts

> - a repo that is the source of truth for JEPs.
> - that repo should be used for rendering of any website content 

👍 I was imagining that the addition I'm suggesting wouldn't change the structure of the current JEP repository, it would probably instead be adding a github action to it + some config that would render the JEPs that are here and push to gh-pages.

> - do we have a process similar to PEP 1 about purpose and guidelines as well as [well written PEPs](https://www.python.org/dev/peps/pep-0001/#id46)

I haven't seen any, but I thought that @captainsafia made a good start here: https://github.com/jupyter/enhancement-proposals/pull/29/files#diff-f6c91f7d040c33d1fb435d5e1b758bedR30

> I think it's important to move [JEP 29](https://github.com/jupyter/enhancement-proposals/pull/29) from draft status to accepted before publishing JEPs on a website.

Now I feel silly, I thought it had already been "accepted" but now I see that it is in "submitted" state. I agree that it makes sense to formalize that process before publishing publicly (since one of the reasons I wanna do this is precisely to highlight that process). I am happy to help with nailing down the process / language in the meta-JEP if others are interested in getting it closer to acceptance (which I guess requires a steering council vote?)

**Comment by damianavila** [comment]
+1 on the idea! Thanks @choldgraf for your continuous effort trying to make visible these sorts of things!

**Comment by willingc** [comment]
Thanks @choldgraf. Perhaps the website would increase the sense of urgency to move JEP 29 to accepted.

**Comment by choldgraf** [comment]
I was looking through [the JEP documentation here](https://github.com/jupyter/enhancement-proposals/blob/master/jupyter-enhancement-proposal-guidelines/jupyter-enhancement-proposal-guidelines.md) and have a couple of points to note:

1. According to [the JEP process described there](https://github.com/jupyter/enhancement-proposals/blob/master/jupyter-enhancement-proposal-guidelines/jupyter-enhancement-proposal-guidelines.md#jep-submission-process) (which is [linked in the readme](https://github.com/jupyter/enhancement-proposals/blob/master/README.md#how-do-i-submit-a-jep), once a PR is merged then it is "Accepted", but not yet "implemented". So considering that [JEP 29 was merged](https://github.com/jupyter/enhancement-proposals/pull/29) I would take that to mean that it is "Accepted" and is just awaiting implementation. Do others agree?
2. The word "completed" is used in [the JEP readme](https://github.com/jupyter/enhancement-proposals/blob/master/README.md#numerical-index), but isn't mentioned in the JEP docs, which I believe instead [use the word `implemented`](https://github.com/jupyter/enhancement-proposals/blob/master/jupyter-enhancement-proposal-guidelines/jupyter-enhancement-proposal-guidelines.md#jep-labels). Are these synonyms?

**Comment by choldgraf** [comment]
OK, I haven't heard any objections to the above, so I am going to consider that @captainsafia's JEP is **accepted** (I believe this is true according to both versions of the JEP rules), and I will try and make a PR **implementing** part of the vision that she lays out. I'll try to get something done soon as a first step towards improving this process.

**Comment by captainsafia** [comment]
Sounds good. Thanks for putting in the work, @choldgraf!

**Comment by blink1073** [comment]
Awesome, thanks @captainsafia and @choldgraf !

**Comment by nealmcb** [comment]
Thanks folks, this sounds good.

I don't know if it has been discussed yet, but I'd like to see clear documentation of when each process change takes place, e.g. the date that a JEP is accepted. E.g. I ran across this JEP: [Cell ID Addition to Notebook Format
](https://jupyter.org/enhancement-proposals/62-cell-id/cell-id.html) and I was confused trying to figure out what its status was. I guess the document itself didn't change after it was accepted, and the sidebar visible there doesn't provide a date for the acceptance. I guess the answer is based on when it was merged as #62, but I'm still not sure.
